boilerplate/app/i18n/i18n.ts.ejs

```diff
@@ -1,4 +1,8 @@
+<% if (props.useExpo) { -%>
+import * as Localization from "expo-localization"
```

Comment:
Since this works in both Expo and non-Expo apps, let's just use this all the time rather than `react-native-localize`.

https://github.com/expo/expo/tree/a3685a7c9208ed0e4afb00c95717442a9c52a2ac/packages/expo-localization#installation-in-bare-react-native-projects

Reply:
Do we want to bring `react-native-unimodules`?

Reply:
Yeah, let's do that, I think it's worthwhile. cc @yulolimum

Reply:
@jamonholmgren and I briefly talked about this and we want to keep the current changes for now, then I'll remove `react-native-localize` and use only `expo-localization` after bringing `react-native-unimodules`.